.php_cs.cache

```diff
@@ -0,0 +1 @@
74,"src\\Browscap\\Data\\DataCollection.php":578493107,"src\\Browscap\\Data\\Engine.php":1785579477,"src\\Browscap\\Data\\Expander.php":-87759782,"src\\Browscap\\Filter\\CustomFilter.php":877695910,"src\\Browscap\\Filter\\FilterInterface.php":1736974322,"src\\Browscap\\Filter\\FullFilter.php":50749593,"src\\Browscap\\Filter\\LiteFilter.php":1350817706,"src\\Browscap\\Filter\\StandardFilter.php":1377865027,"src\\Browscap\\Generator\\AbstractBuildGenerator.php":1609837931,"src\\Browscap\\Generator\\BuildCustomFileGenerator.php":-1924528179,"src\\Browscap\\Helper\\CollectionCreator.php":-1479446052,"src\\Browscap\\Helper\\LoggerHelper.php":1478676691,"src\\Browscap\\Writer\\CsvWriter.php":729320949,"src\\Browscap\\Writer\\Factory\\CustomWriterFactory.php":842770640,"src\\Browscap\\Writer\\Factory\\FullCollectionFactory.php":-115329305,"src\\Browscap\\Writer\\Factory\\FullPhpWriterFactory.php":352761445,"src\\Browscap\\Writer\\Factory\\PhpWriterFactory.php":470763055,"src\\Browscap\\Writer\\IniWriter.php":-378186785,"src\\Browscap\\Writer\\JsonWriter.php":1270684446,"src\\Browscap\\Writer\\WriterCollection.php":1018323874,"src\\Browscap\\Writer\\WriterInterface.php":89577198,"src\\Browscap\\Writer\\XmlWriter.php":58153522,"tests\\BrowscapTest\\Data\\DataCollectionTest.php":-869545266,"tests\\BrowscapTest\\Data\\ExpanderTest.php":1663945222,"tests\\BrowscapTest\\Data\\Factory\\PlatformFactoryTest.php":-1070710833,"tests\\BrowscapTest\\Filter\\CustomFilterTest.php":248302453,"tests\\BrowscapTest\\Filter\\FullFilterTest.php":1908486573,"tests\\BrowscapTest\\Filter\\LiteFilterTest.php":-364580284,"tests\\BrowscapTest\\Filter\\StandardFilterTest.php":1550463854,"tests\\BrowscapTest\\Formatter\\AspFormatterTest.php":-1970525497,"tests\\BrowscapTest\\Formatter\\CsvFormatterTest.php":257105995,"tests\\BrowscapTest\\Formatter\\JsonFormatterTest.php":-784186953,"tests\\BrowscapTest\\Formatter\\PhpFormatterTest.php":-391938518,"tests\\BrowscapTest\\Formatter\\XmlFormatterTest.php":-2128263355,"tests\\BrowscapTest\\Generator\\BuildCustomFileGeneratorTest.php":461253061,"tests\\BrowscapTest\\Generator\\BuildGeneratorTest.php":-570740156,"tests\\BrowscapTest\\Generator\\Helper\\BuildHelperTest.php":-646473112,"tests\\BrowscapTest\\Helper\\CollectionCreatorTest.php":-1370076622,"tests\\BrowscapTest\\Writer\\CsvWriterTest.php":1101898372,"tests\\BrowscapTest\\Writer\\Factory\\CustomWriterFactoryTest.php":983023773,"tests\\BrowscapTest\\Writer\\Factory\\FullCollectionFactoryTest.php":-1713893646,"tests\\BrowscapTest\\Writer\\Factory\\FullPhpWriterFactoryTest.php":-1270777302,"tests\\BrowscapTest\\Writer\\IniWriterTest.php":-793353956,"tests\\BrowscapTest\\Writer\\JsonWriterTest.php":361816348,"tests\\BrowscapTest\\Writer\\WriterCollectionTest.php":1429067174,"tests\\BrowscapTest\\Writer\\XmlWriterTest.php":-1628250298,"tests\\UserAgentsTest\\FullTest.php":1551869573,"tests\\UserAgentsTest\\LiteTest.php":-459569176,"tests\\UserAgentsTest\\StandardTest.php":727140160}}
```

Comment:
This file can cause weird merge conflict issues when merging/rebasing, fyi.

It's nice to have, because it speeds up the php-cs-fixer run quite a bit, but it effectively has to be wiped and re-built if it's ever in a merge conflict.

We haven't figured out a good way to deal with this, and I'm not sure there is other than wiping/rebuilding on conflict, or not committing at all.

How do other open source projects deal with this?

Reply:
I can remove this file and add it to .gitignore.

Reply:
I have seens some projects which do have this file in the repo, but others have. But I dont know how they handle conflicts.

Reply:
Do you think it's important to have the cache file for Travis?

Reply:
file removed and ignored